lib/matplotlib/figure.py

```diff
@@ -331,8 +331,8 @@ def _suplabels(self, t, info, **kwargs):
         fontproperties : None or dict, optional
             A dict of font properties. If *fontproperties* is given the
             default values for font size and weight are taken from the
```

Comment:
```suggestion
            default, values for font size and weight are taken from the
```

Reply:
Maybe `If *fontproperties* is not given, the ...`?

Or maybe it is correct, just that I have problem understanding?